tests/data/synthetic pre2018 ROI.dat

```diff
@@ -0,0 +1,3 @@
+ROI index	ROI ID	ROI Time (ns)	Pixels in ROI	X start	Y start	Z start	X stop	Y stop	Z stop	Angle (deg)	Composite ID	Number of lines	Frame Size	Zoom	Laser Power (%)	ROI group ID
+1	0	4000	1	100	50	10	110	54	10	0	0	0	512	1	44.444	0
+2	0	4000	1	150	60	10	160	64	10	0	0	0	512	1	44.444	0
```

Comment:
Maybe I'm reading this wrong, but is the pixel number 1 according to this? I suggest keeping the file as consistent or "realistic" as possible (so 40 pixels in this case), even if it idoesn't matter for the tests -- and it may in the future!